### DISCUSSION TITLE
Display Raspberry Pi Temp

### DISCUSSION THREAD

**Discussion by Hect0rG** [tagyoureit/nodejs-poolController#417] [Q&A]
Is there a way to display the raspberry pi CPU temp on the main screen and maybe call it Controller Temp. I don't have a solar pool heater so that would be the perfect spot. 

I already have it on Generic Devices on REM just not sure how to expose it. 

![image](https://user-images.githubusercontent.com/6274208/163695368-95bd712f-e045-4a53-9c25-81fcd18c8e40.png

**Comment by rstrouse** [comment]
Let me think about some possibilities with this.  The solar temp location is probably not the right spot for it when solar equipment is installed.  I have been thinking about ways to create custom panels.  In the meantime, do you have a fan on your pi?  117F is 47C and the pi will start to throttle at 50C.

**Reply by Hect0rG** [reply]
I do have an argon fan hat on but I think I had it off when I took that screenshot. I have this mounted outdoors so it does probably get that hot from time to time which is exactly why I want to have that temp visible.

**Reply by rstrouse** [reply]
If it was off then 47C is probably about right.  Oddly enough if you blow 110F air over the chip then it will be plenty cool enough.
